project/src/main/webapp/script.js

```diff
@@ -23,8 +23,8 @@ window.onload = function onLoad() {
 async function checkLoginStatus() {
   const response = await fetch('/login-status');
   const loginStatus = await response.json();
-  const isLoggedIn = loginStatus.isLoggedIn;
-  if (isLoggedIn) {
+  const loginState = loginStatus.loginState;
+  if (loginState === 'LOGGED_IN') {
```

Comment:
nit: Use the enum

Reply:
Ack - This is in the javascript maybe when we make a common.js for the javascript we can add this as a constant to the file.